code-challenges.js

```diff
@@ -8,32 +8,115 @@
 // --------------------1) Create a statement that determines if a temperature is below boiling point, at boiling point, or above boiling point. Boiling point is 212 degrees Fahrenheit.
 
 // Create the code and test each of the variables provided. Expected output: "42 is below boiling point", "350 is above boiling point", "212 is at boiling point"
-
-var temp = 42
-// var temp = 350
-// var temp = 212
+//
+var temp = 42;
+
+if (temp === 212) {
+  console.log( temp +  ' is at boiling point');
+} else if (temp < 212) {
+  console.log( temp + ' is below boiling point');
+} else {
+  console.log( temp + ' is above boiling point');
+}
+
+//I was initially getting the following error mesage "console.log(temp ' is at boiling point');
+//SyntaxError: missing ) after argument list"
+// relaixed I forgot to add "+" after the variable temp in the console log.
+
+var temp = 350
+
+if (temp === 212) {
+  console.log( temp +  ' is at boiling point');
+} else if (temp < 212) {
+  console.log( temp + ' is below boiling point');
+} else {
+  console.log( temp + ' is above boiling point');
+}
+
+var temp = 212
+
+if (temp === 212) {
+  console.log( temp +  ' is at boiling point');
+} else if (temp < 212) {
+  console.log( temp + ' is below boiling point');
+} else {
+  console.log( temp + ' is above boiling point');
+}
```

Comment:
Looks good. You only needed one set of code. You just want to test it with the different variables.